### DISCUSSION TITLE
How to attach image to the e-mail?

### DISCUSSION THREAD

**Discussion by iBobik** [Dave136/vue-email#204] [Q&A]
When remote image is used most clients will block them to prevent tracking.
```
<Img src="https://example.com/images/logo.png" width="192" height="98" />
```

How to make the image field attached in the e-mail so it will be rendered?

**Comment by gabalicious** [comment]
There is a img component but also I think your img tag above has uppercase I which isn't valid.

https://vue-email.vercel.app/components/image
